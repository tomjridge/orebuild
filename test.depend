p1_lib.cmi :
p1_parse_grammar.cmi : p1_lib.cmi
interactive.cmo :
interactive.cmx :
p1_cl.cmo : p1_lib.cmi p1_extra_combinators.cmo
p1_cl.cmx : p1_lib.cmx p1_extra_combinators.cmx
p1_core.cmo : p1_prelude.cmo
p1_core.cmx : p1_prelude.cmx
p1_examples.cmo : p1_lib.cmi
p1_examples.cmx : p1_lib.cmx
p1_extra.cmo : p1_core.cmo
p1_extra.cmx : p1_core.cmx
p1_extra_combinators.cmo : p1_lib.cmi
p1_extra_combinators.cmx : p1_lib.cmx
p1_gen.cmo : p1_parse_grammar.cmi p1_lib.cmi p1_gen_shared.cmo p1_extra_combinators.cmo p1_cl.cmo
p1_gen.cmx : p1_parse_grammar.cmx p1_lib.cmx p1_gen_shared.cmx p1_extra_combinators.cmx p1_cl.cmx
p1_gen_p4.cmo : p1_parse_grammar.cmi p1_lib.cmi p1_gen_shared.cmo p1_extra_combinators.cmo p1_cl.cmo
p1_gen_p4.cmx : p1_parse_grammar.cmx p1_lib.cmx p1_gen_shared.cmx p1_extra_combinators.cmx p1_cl.cmx
p1_gen_shared.cmo : p1_parse_grammar.cmi p1_lib.cmi p1_extra_combinators.cmo p1_cl.cmo
p1_gen_shared.cmx : p1_parse_grammar.cmx p1_lib.cmx p1_extra_combinators.cmx p1_cl.cmx
p1_lib.cmo : p1_util.cmo p1_parsers.cmo p1_core.cmo p1_lib.cmi
p1_lib.cmx : p1_util.cmx p1_parsers.cmx p1_core.cmx p1_lib.cmi
p1_parse_grammar.cmo : p1_lib.cmi p1_extra_combinators.cmo p1_parse_grammar.cmi
p1_parse_grammar.cmx : p1_lib.cmx p1_extra_combinators.cmx p1_parse_grammar.cmi
p1_parsers.cmo : p1_core.cmo
p1_parsers.cmx : p1_core.cmx
p1_prelude.cmo :
p1_prelude.cmx :
p1_util.cmo :
p1_util.cmx :
