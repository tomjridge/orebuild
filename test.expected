p1_lib.mli p1_parse_grammar.mli p1_extra_combinators.ml p1_parse_grammar.ml p1_lib.ml p1_cl.ml p1_gen_shared.ml p1_gen_p4.ml p1_gen.ml p1_examples.ml
